src/enchant.1.in

```diff
@@ -38,13 +38,13 @@ The ordering file takes the form language_tag:<comma-separated list of spelling
 providers>. To see what providers are available, run \fIenchant-lsmod-@ENCHANT_MAJOR_VERSION@\fR. '*' is
 used to mean "use this ordering for all languages, unless instructed otherwise." For example:
 .PP
-*:aspell,hunspell
+*:nuspell,aspell,hunspell
```

Comment:
In this and all other example orderings, please place nuspell after hunspell.

Reply:
https://github.com/nuspell/enchant/commit/1739563777b88c5cb0203bb33ca77d784ca06d8b#diff-59b29c14ce81e4df9a0a56a362f52d27R41